backends/tfhe-cuda-backend/cuda/src/integer/compression/compression.cu

```diff
@@ -25,24 +25,25 @@ void scratch_cuda_integer_decompress_radix_ciphertext_64(
     uint32_t compression_glwe_dimension, uint32_t compression_polynomial_size,
     uint32_t lwe_dimension, uint32_t pbs_level, uint32_t pbs_base_log,
     uint32_t num_lwes, uint32_t message_modulus, uint32_t carry_modulus,
-    PBS_TYPE pbs_type, uint32_t storage_log_modulus, bool allocate_gpu_memory) {
+    PBS_TYPE pbs_type, uint32_t storage_log_modulus, uint32_t bodies_count,
```

Comment:
Is bodies_count used? I doesn't seem so to me? :thinking:

Reply:
Yes, we need it to unpack the input GLWE in decompression. I was initially trying to compute it inside the function but I concluded it was easier to just pass it from rust side.

However there is a inconsistency in naming. Some places we call it "bodies" and in other "body". I will refactor.